tools/prepush.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 # This script is configured to be run by git pre-push
 
-cd rust/template/
-cargo fmt --all -- --check
-cargo check --all
-cd ..
+(cd rust/template/ && \
+cargo fmt --all -- --check && \
```

Comment:
Thank you!
You may also want to include
```
cargo clippy --all -- -D warnings
```

Reply:
Can't run `cargo` in `lib`, as there is no `Cargo.toml` there.

Reply:
Sure, but my comment was about `rust/template` (I missed when you introduced the file). Anyway, probably no need to schedule yet another pipeline...

Reply:
Oh, I see. I'll add it to my next PR.